repair/repair.cc

```diff
@@ -71,17 +71,17 @@ node_ops_metrics::node_ops_metrics(shared_ptr<repair::task_manager_module> modul
     auto ops_label_type = sm::label("ops");
     _metrics.add_group("node_ops", {
         sm::make_gauge("finished_percentage", [this] { return bootstrap_finished_percentage(); },
-                sm::description("Finished percentage of node operation on this shard"), {ops_label_type("bootstrap")}),
+                scylla_node_ops_finished_percentage , {ops_label_type("bootstrap")}),
```

Comment:
I presume that you'll want the name "scylla_node_ops_finished_percentage" to be exactly the name of the metric, because the documentation code that will go over the header file will otherwise not know the name of the metric. So, I think you should have some sort of automatic code which verifies that the name "scylla_node_ops_finished_percentage" is really the name of the metric being created.

Reply:
Theoretically, I can add some kind of validation, but it will be in run time (see general note ```d```)